build/BuildFunctions.psm1

```diff
@@ -111,96 +111,96 @@ function CompileTask($task) {
     }
 }
 
-function TestAll([string[]]$tasks) {
+function Invoke-AllMochaTests([string[]]$tasks, [string]$reporter) {
     Write-Host "Testing Tasks"
     $jobs = $tasks | % {
-        Start-Job -ArgumentList $pwd, $_ -ScriptBlock {
+        Start-Job -ArgumentList $pwd, $_, $reporter -ScriptBlock {
             cd $args[0]
             Import-Module ./build/BuildFunctions.psm1
-            TestTask $args[1]
+            Invoke-MochaTest -task $args[1] -reporter $args[2]
```

Comment:
Those args are to the script block and come from the `-ArgumentList` parameter three lines above.